tests/unit/test_sqs.py

```diff
@@ -175,3 +176,20 @@ def test_is_sqs_queue_url():
     assert is_sqs_queue_url("http://us-east-1.queue.foo.bar:4566/111111111111/foo") is True
     assert is_sqs_queue_url("http://queue.foo.bar:4566/111111111111/foo") is True
     assert is_sqs_queue_url("http://sqs.us-east-1.foo.bar:4566/111111111111/foo") is True
+
```

Comment:
It would probably good if we could have integration tests for this as well. Not blocking for now, but having tests that set the endpoint like our customers might become necessary at some point